### PR TITLE
Switch to "get this [media]" for single result CTAs

### PR DESCRIPTION
**Pull request by zackkrida**
## Fixes

Related to #1701 by @panchovm 

## Description

Switches to the agreed-upon "get this" version of our single result CTAs.

## Checklist

- [x] My pull request has a descriptive title (not a vague title like `Update index.md`).
- [x] My pull request targets the *default* branch of the repository (`main`) or a parent feature branch.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [x] I added or updated tests for the changes I made (if applicable).
- [x] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no visible errors.

[best_practices]:https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

**Review by krysal — APPROVED** [review]

**Review by panchovm — COMMENTED** [review]
It looks good.

I would add the external icon to be explicit about landing on a different site. I will update the design issue to specify the icon and the spacing.

Are we also updating the component?

**Comment by panchovm** [comment]
Here is [the presentation](https://www.figma.com/proto/ClEKESZzZxudMoHbHq3A66/Copy-improvements-in-the-search-flow?page-id=26%3A13657&node-id=427%3A9251&viewport=461%2C537%2C0.3&scaling=min-zoom&starting-point-node-id=427%3A9251) where I put the three changes. For this case, we need to show the external icon in both image and audio details

**Comment by obulat** [comment]
I've opened a PR to this PR adding the external link icon, but it has the `aaargggghhh` linting errors, so I don't know what to do with it. I really hope we don't re-introduce them.
https://github.com/WordPress/openverse-frontend/pull/1755

File: test/playwright/playwright.config.ts [hunk 1]
```diff
@@ -52,6 +52,12 @@ const config: PlaywrightTestConfig = {
    * Playwright default of using 1/2 of the number of CPU cores continues to work otherwise.
    */
   workers: UPDATE_TAPES === 'true' ? 1 : undefined,
+  expect: {
+    toMatchSnapshot: {
+      // To avoid flaky tests, we allow a small amount of pixel difference.
+      maxDiffPixelRatio: 0.01,
```
Comment (krysal) [inline test/playwright/playwright.config.ts:58]: Nice addition 👌

**Review by krysal — APPROVED** [review]

**Pull request merged**